### ISSUE TITLE
Name of the subfolder in GECKO/tutorials/

### ISSUE THREAD

**Issue by Yu-sysbio** [SysBioChalmers/GECKO#330]
Considering that the comparison between light and full is now on the yeast model, I am wondering if it is needed to modify the names of the subfolders in the [path](https://github.com/SysBioChalmers/GECKO/tree/develop/tutorials), maybe like:

> full_ecModel --> yeast_ecModel (or yeast, yeast_case, etc)
> light_ecModel --> human_ecModel (or human, human_case, etc)

Or just keep it as is otherwise we have to modify throughout the manuscript.

**Comment by edkerk** [comment]
There's good reason why we removed `yeast` and `human` from the tutorial names. Making some adjustments in the protocol and clarification that the _light_ vs. _full_ comparison is for yeast-only should help. It's also for that purpose that I've packaged the whole comparison in one function, the user will not see the _light_ ecModel when running it, only the light_ecModel tutorial gives the user a _light_ ecModel to play around with.
